<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>brickgame</class>
 <widget class="QMainWindow" name="brickgame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>brickgame</string>
  </property>
  <property name="iconSize">
   <size>
    <width>45</width>
    <height>45</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="ButtonSnake">
      <property name="minimumSize">
       <size>
        <width>374</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: black;
    color: white;
    border: 2px solid gold;
    border-radius: 10px;
    font-family: &quot;Snake Chan&quot;;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: darkgoldenrod;
    color: black;
}

QPushButton:pressed {
    background-color: gold;
    color: white;
    border: 2px solid darkred;
}
</string>
      </property>
      <property name="text">
       <string>SNAKE</string>
      </property>
      <property name="icon">
       <iconset theme="accessories-calculator">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonTetris">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: black;
    color: white;
    border: 2px solid gold;
    border-radius: 10px;
   font-family: 'Courier New', monospace;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: darkgoldenrod;
    color: black;
}

QPushButton:pressed {
    background-color: gold;
    color: white;
    border: 2px solid darkred;
}
</string>
      </property>
      <property name="text">
       <string>TETRIS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonExit">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: black;
    color: white;
    border: 2px solid red;
    border-radius: 10px;
    font-family: &quot;Impact&quot;, &quot;Arial Black&quot;, sans-serif;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: red;
    color: white;
}

QPushButton:pressed {
    background-color: black;
    color: white;
    border: 2px solid darkred;
}
</string>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
